package.json

```diff
@@ -45,49 +45,54 @@
     "grunt-jsbeautifier": "^0.2.13",
     "grunt-jscs": "^3.0.1",
     "mkdirp": "^0.5.1",
-    "nyc": "^8.3.2",
-    "sinon": "^1.17.7"
+    "nyc": "^11.1.0",
+    "sinon": "^3.1.0"
   },
   "dependencies": {
-    "async": "^1.4.2",
+    "async": "^2.5.0",
     "bindings": "^1.2.1",
     "block-stream2": "^1.1.0",
     "colors": "^1.1.0",
     "common-tags": "^1.4.0",
-    "fs-extra": "^0.26.2",
+    "fs-extra": "^4.0.1",
     "fs-temp": "^1.0.0",
     "fstream": "^1.0.8",
     "fstream-ignore": "1.0.5",
-    "glob": "^5.0.15",
-    "inquirer": "^0.8.5",
+    "glob": "^7.1.2",
+    "inquirer": "^3.2.1",
     "is-root": "^1.0.0",
     "mdns-js": "^0.5.0",
     "minimatch": "^3.0.0",
-    "node-rsa": "^0.2.26",
+    "node-rsa": "^0.4.2",
     "nomnom": "^1.8.1",
-    "npm": "^3.10.10",
+    "npm": "^5.3.0",
```

Comment:
Can this cause problems ?

Reply:
It may, but I haven't finished the greenkeeper testing so I have no idea yet :)